src/EUTelGeometryTelescopeGeoDescription.cpp

```diff
@@ -883,14 +941,105 @@ const TGeoHMatrix* EUTelGeometryTelescopeGeoDescription::getHMatrix( const doubl
  * 
  * @return radiation length in units of X0
  */
-float EUTelGeometryTelescopeGeoDescription::findRadLengthIntegral( const double globalPosStart[], const double globalPosFinish[], bool skipBoundaryPonitsVolumes ) {
 
+float EUTelGeometryTelescopeGeoDescription::findRad( const double globalPosStart[], const double globalPosFinish[], bool skipBoundaryPonitsVolumes, 	std::map< const int, double> &sensors, 	std::map< const int, double> &air ){
```

Comment:
can this (and similar functions) be moved to EUTelNav or a new class? EUTelGeometryTelescopeGeoDescription is a class dedicated to describing the setup, i.e. it has a state (e.g. where sensorX is located). I'll send you a slide from richard's presentation